core/src/main/resources/volcano-viz/volcano-viz-data.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample generated visualizer data used by the visualizer JavaScript code
+var data = {
+
+	allNodes: {
+		"node1": {
+			label: "TableScan-1", // node label being displayed
+			isSubset: false, // if the node is a RelSubset
+			explanation: "table=t1", // additional explanation of properties
+			finalCost: "100 cpu, 100io" // final cost (in string) of the node at the end of optimization
+		},
+		"node2": {
+			label: "Filter-2",
+			isSubset: false,
+			explanation: "condition=c",
+			finalCost: "200cpu, 100io"
+		},
+		"node3": {
+			label: "TableSink-3",
+			isSubset: false,
+			explanation: "table=t2",
+			finalCost: "20cpu, 20io"
+		},
+		"node4": {
+			label: "IndexTableScan-4",
+			isSubset: false,
+			explanation: "table=t1, condition=c",
+			finalCost: "10cpu, 10io"
+		},
+	},
+
+	ruleMatchSequence: [
+		"INITIAL",
+		"IndexTableScanRule#1",
+		"FINAL"
+	],
+
+	ruleMatchInfoMap: {
+		"INITIAL": {
+			setLabels: {
+				"set1": "set1",
+				"set2": "set2",
+				"set3": "set3",
+			},
+			setOriginalRel: {
+				"set1": "node1",
+				"set2": "node2",
+				"set3": "node3",
+			},
```

Comment:
The setLabels and setOriginalRel could be moved to the root level (i.e., to line 47). This requires changing the js code, though.

Reply:
agree, this can make the json more compact. I can do the JS changes since I'm more familiar.